benchmark/opperf/utils/op_registry_utils.py

```diff
@@ -122,12 +122,21 @@ def prepare_op_inputs(op, arg_params):
     # 3d tensor is needed by following ops
     ops_3d = ['CTCLoss', 'ctc_loss']
 
+    custom_data = ['SpatialTransformer', 'col2im', 'RNN', 'GroupNorm', 'Dropout', 'FullyConnected', 'SoftmaxOutput', 'LinearRegressionOutput', 'BatchNorm',
+                            'LogisticRegressionOutput', 'MAERegressionOutput', 'SVMOutput', 'L2Normalization',
+                            'LayerNorm', 'InstanceNorm', 'Embedding', 'Correlation', 'im2col', 'LRN']
+
+    # Args for NN basic ops which vary across different ops
+    ops_nn_variable_args = ['weight', 'label', 'mode', 'gamma', 'beta', 'kernel', 'stride', 'dilate', 'pad', 'p', 'axis', 'loc', "moving_mean","moving_var"]
```

Comment:
why isn't this variable `ops_nn_variable_args` used anywhere else?

Reply:
Good point, dropping.